energy-growers/src/components/Footer/Footer.scss

```
@@ -2,29 +2,96 @@
 
 .eg-footer {
   width: 100vw;
-  padding-top: rem(72);
   overflow: hidden;
+  position: relative;
 
   .eg-footer-background {
     z-index: -1;
     object-fit: cover;
   }
-}
 
-.eg-sub-footer {
-  width: 100%;
-  padding-top: rem(151);
+  .eg-sub-menu {
+    padding-top: rem(51);
+    padding-bottom: rem(33);
+
+    @media screen and (min-width: rem(map-get($breakpoints, sm))) {
+      padding-top: rem(58);
+      padding-bottom: rem(45);
+    }
+
+    @media screen and (min-width: rem(map-get($breakpoints, md))) {
+      padding-top: rem(81);
+      padding-bottom: rem(129);
+    }
+
+    @media screen and (min-width: rem(map-get($breakpoints, lg))) {
+      padding-top: rem(85);
+      padding-bottom: rem(89);
+    }
+
+    @media screen and (min-width: rem(map-get($breakpoints, xl))) {
+      padding-top: rem(118);
+      padding-bottom: rem(129);
+    }
+
+    @media screen and (min-width: rem(map-get($breakpoints, xxl))) {
+      padding-top: rem(121);
+      padding-bottom: rem(159);
+    }
```

```suggestion
    padding-bottom: rem(33);

    @media screen and (min-width: rem(map-get($breakpoints, sm))) {
      padding-top: rem(58);
      padding-bottom: rem(45);
    }

    @media screen and (min-width: rem(map-get($breakpoints, md))) {
      padding-top: rem(81);
      padding-bottom: rem(129);
    }

    @media screen and (min-width: rem(map-get($breakpoints, lg))) {
      padding-top: rem(85);
      padding-bottom: rem(89);
    }

    @media screen and (min-width: rem(map-get($breakpoints, xl))) {
      padding-top: rem(118);
      padding-bottom: rem(129);
    }

    @media screen and (min-width: rem(map-get($breakpoints, xxl))) {
      padding-top: rem(121);
      padding-bottom: rem(159);
    }
```

i padding bottom non campaciano perche nella lista dei link c'è un padding bottom,
o lo annulli nell'ultimo figlio o vedi tu